libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteRefreshEx.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.navigation.base.route
```

Comment:
Could we move this under the `internal.extensions` package? This way CI (Dokka) wouldn't complain https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/8378/workflows/fe1e6a0c-931e-45f2-ae62-6ac51ff43f5f/jobs/31149 😅